site/soc/software/charts/osh/openstack-keystone/keystone.yaml

```diff
@@ -58,4 +58,22 @@ data:
             Allow from all
           </IfModule>
         </Directory>
+    bootstrap:
+      # We override this due to SOC-9409: admin role not assigned to admin user on the default domain
+      # upstream patch: https://review.opendev.org/662993
+      # TODO(itxaka): remove once upstream patch is merged and airship git sha for openstack-helm updated
+      script: |
+        #NOTE(gagehugo): As of Rocky, keystone creates a member role by default
+        openstack role create --or-show member
+        openstack role add \
+              --user="${OS_USERNAME}" \
+              --user-domain="${OS_USER_DOMAIN_NAME}" \
+              --project-domain="${OS_PROJECT_DOMAIN_NAME}" \
+              --project="${OS_PROJECT_NAME}" \
+              "member"
```

Comment:
BTW as we _are using rocky_, you could remove this member role addition (assuming the comment above from gagehugo is true)

Reply:
mmh, good point, but I wanted it to be as similar to upstream as possible to not trigger any unknown condition, because we all know comments never lie ;)